plugin/pkg/auth/authorizer/rbac/bootstrappolicy/namespace_policy.go

```diff
@@ -93,6 +93,15 @@ func init() {
 			eventsRule(),
 		},
 	})
+	addNamespaceRole(metav1.NamespaceSystem, rbac.Role{
+		// role for the leader locking on supplied configmap
+		ObjectMeta: metav1.ObjectMeta{Name: "system::leader-locking"},
+		Rules: []rbac.PolicyRule{
```

Comment:
@deads2k looking for .Names("kube-controller-manager", "kube-scheduler") ?  

or are you looking for separate rules for each one?   I'm new to this area of the code so this is also a learning experience for me atm.